src/com/goide/psi/GoFile.java

```diff
@@ -414,7 +415,11 @@ public String getPackageName() {
     if (packageClause != null) {
       PsiElement packageIdentifier = packageClause.getIdentifier();
       if (packageIdentifier != null) {
-        return packageIdentifier.getText().trim();
+        String name = packageIdentifier.getText().trim();
+        if (GoTestFinder.isTestFile(this)) {
```

Comment:
1. could be rewritten with ternary operator (not required)
2. the case when isTestFile(file)=false and package name ends with _test is not covered with test
3. the case when isTestFile(file)=true and package name doesn't end with _test is not covered with test